content/plugin/log/filtering.mdx

```diff
@@ -0,0 +1,73 @@
+---
+page_title: 'Home - Plugin Development: Filtering Log Output'
+description: >-
+  Information on how to filter the logs that your provider outputs, to help
+  find the information you're looking for.
+---
+
+# Filtering Log Output
+
+A lot of different pieces of software are included in a Terraform run: Terraform itself, the SDK or SDKs that providers are built on, and one or more providers. All of these disparate pieces of software can be writing logs, which means that a lot of different logs are being written, and not all of them are always useful in figuring out why something's happening. To help with that, there are various environment variables used to filter log output.
+
+## Valid Levels
+
+Valid levels you can pass to these environment variables are as follows, in order of most verbose to least verbose:
+
+* `TRACE`
+* `DEBUG`
+* `INFO`
+* `WARN`
+* `ERROR`
+* `OFF` (turns off logs for that logger)
```

Comment:
Can we do more here to explain what people will get when they set these levels? It's hard right now for me to understand which one I would want to pick. Can we have a brief description with each one about what users are going to get?  

Also - does JSON work with every single one of the things below? Can you set something to both JSON and one of the other levels? Or is it the case that if you set it to JSON you automatically get the most verbose thing?